enos/enos-scenario-ui.hcl

```diff
@@ -57,49 +62,57 @@ scenario "ui" {
   }
 
   step "ec2_info" {
-    module = module.ec2_info
+    description = global.description.ec2_info
+    module      = module.ec2_info
   }
 
   step "create_vpc" {
-    module = module.create_vpc
+    description = global.description.create_vpc
+    module      = module.create_vpc
 
     variables {
       common_tags = local.tags
     }
   }
 
-  step "create_seal_key" {
-    module = "seal_${local.seal}"
-
-    variables {
-      cluster_id  = step.create_vpc.cluster_id
-      common_tags = global.tags
-    }
-  }
-
   // This step reads the contents of the backend license if we're using a Consul backend and
   // the edition is "ent".
   step "read_backend_license" {
-    skip_step = matrix.backend == "raft" || var.backend_edition == "ce"
-    module    = module.read_license
+    description = global.description.read_backend_license
+    skip_step   = matrix.backend == "raft" || var.backend_edition == "ce"
+    module      = module.read_license
 
     variables {
       file_name = local.backend_license_path
     }
   }
+  // TODO: vault enterprise requires a license
+  // TODO: vault CE does not require a license
```

Comment:
Are these meant to be included in this PR, or later?

Reply:
That was just a reminder for me to do a quality, which exists as `"vault_requires_license_for_enterprise_editions"`